value/value.md

```diff
@@ -231,87 +225,183 @@ the player to your app, and for which nothing already exists, add your own.
+- `C` will receive $\dfrac{1}{100 + 99 + 10} = 0.005 = 0.5\\%$ of the payment
 
-If, instead of a 50/40/10 (total of 100) split, the splits were given as 190/152/38 (total of 380), the respective
-payment amounts each minute would still
-be 50 sats, 40 sats and 10 sats because the listener chose to pay 100 sats per minute, and the respective shares (as a
-percentage of the total) would remain the same.
+This example demonstrates why splits should not be required to add up to 100—it allows for recipients to receive payments that are not just integer percent values.
+In this specific case, it allows `B` to receive $49.5\\%$ and `C` to receive $0.5\\%$.
 
-On a 190/152/38 split, each minute the payment calculation would be:
+Suppose that the listener chose to send a 1000-sat boost, which the app decides to distribute immediately among the recipients:
 
-- Interval payout: 100 sats
+Batch payout: **1000 sats**
+  
+- Recipient `A` gets a payment of 500 sats (calculated using $1000 \cdot \dfrac{100}{100 + 99 + 1} = 500$)
+- Recipient `B` gets a payment of 495 sats (calculated using $1000 \cdot \dfrac{99}{100 + 99 + 1} = 495$)
+- Recipient `C` gets a payment of 5 sats (calculated using $1000 \cdot \dfrac{1}{100 + 99 + 1} = 5$)
 
-- Share total: 380
+#### Example 3: recipients with `fee=true`
 
-    - Recipient #1 gets a payment of: 50 sats (190 / 380 * 100)
-    - Recipient #2 gets a payment of: 40 sats (152 / 380 * 100)
-    - Recipient #3 gets a payment of: 10 sats (38 / 380 * 100)
+```xml
+
+<podcast:value type="lightning" method="keysend" suggested="0.00000015000">
+    <podcast:valueRecipient
+            name="A"
+            type="node"
+            address="02d5c1bf8b940dc9cadca86d1b0a3c37fbe39cee4c7e839e33bef9174531d27f52"
+            split="101"
+    />
+    <podcast:valueRecipient
+            name="B"
+            type="node"
+            address="032f4ffbbafffbe51726ad3c164a3d0d37ec27bc67b29a159b0f49ae8ac21b8508"
+            split="99"
+    />
+    <podcast:valueRecipient
+            name="C"
+            type="node"
+            address="03ae9f91a0cb8ff43840e3c322c4c61f019d8c1c3cea15a25cfc425ac605e61a4a"
+            split="16"
+            fee="true"
+    />
+    <podcast:valueRecipient
+            name="D"
+            type="node"
+            address="039e2bd74dadb4a50c3be632a5068cbc820d520635acc5c527a2171ab6cab43a9c"
+            split="4"
+            fee="true"
+    />
+</podcast:value>
+```
 
-If an app chooses to only make a payout once every 30 minutes of listening/watching, the calculation would be the same
-after multiplying
-the per-minute payment by 30:
+In this case, there are two `fee=true` recipients, so the calculation will have to be done in two steps:
 
-- Interval payout: 3000 sats (100 * 30)
+1. `C` will receive $16\\%$ and `D` will receive $4\\%$ off the top of each payment.
+2. The rest of the payment (i.e., $100\\% - 16\\% - 4\\% = 80\\%$) will be distributed among `A` and `B` in the ratio $101 : 99$.
```

Comment:
```suggestion
2. The rest of the payment (i.e., $100\\% - (16\\% + 4\\%) = 80\\%$) will be distributed among `A` and `B` in the ratio $101 : 99$.
```
Just to help clarify that fee percentages are deducted from the total in subsequent steps (as per my question in the linked issue).